optimade/server/mappers/entries.py

```diff
@@ -34,7 +32,7 @@ def alias_for(cls, field: str) -> str:
         """
         return dict(cls.all_aliases()).get(field, field)
 
-    @abc.abstractclassmethod
+    @classmethod
     def map_back(cls, doc: dict) -> dict:
         """Map properties from MongoDB to OPTiMaDe
```

Comment:
Could you add to this docstring to just explain what this method does, and briefly how it might be extended/wrapped by subclasses? e.g. takes a Mongo document, applies any implementation-specific aliases, populates attributes with the stored data, scrubbing any conflicting top-level fields, ...

Reply:
I can try. If you have any good explanations you are more than welcome to add one as well :+1:

Reply:
Sure, happy to add once you're done with all the features. We probably want to do a big docs update at some point too...